libheif/api/libheif/heif_properties.cc

```diff
@@ -409,3 +411,29 @@ struct heif_error heif_item_get_property_raw_data(const struct heif_context* con
 
   return heif_error_success;
 }
+
+struct heif_error heif_item_get_property_extended_type(const struct heif_context* context,
```

Comment:
I know that it is called `extended_type` in the standard. But what do you think about calling it `heif_item_get_property_uuid_type`? I think that makes it clearer to the use who does not know the standard by heart that this belongs together to `uuid` boxes.

Reply:
I can do that. I considered putting uuid in the name, so happy with the suggestion.